src/components/maincontroller.js

```diff
@@ -184,15 +179,10 @@ mgr.addEventListener(cast.framework.events.EventType.ENDED, function () {
     reportPlaybackStopped($scope, getReportingParams($scope));
     init();
 
-    if (!playNextItem()) {
+    if (!playbackMgr.playNextItem()) {
```

Comment:
This got caught by the linter, playNextItem was not defined, so this would break playing the next item in the queue I think.